paddle/fluid/operators/sgd_op.h

```diff
@@ -47,6 +47,11 @@ class SGDOpKernel : public framework::OpKernel<T> {
       PADDLE_ENFORCE_EQ(param, param_out);
       auto* grad = ctx.Input<framework::SelectedRows>("Grad");
 
+      // if sparse variable is not initialized, skip update
```

Comment:
Need more detailed comments. I remember this is because pserver may receive empty `SelectedRows` variable splited by trainers.

Reply:
Will do that, pserver wouldn't receive empty `SelectedRows` currently, https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/fluid/operators/send_op.cc#L70 .

Reply:
Done.